public/index.html

```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Why are we removing this tag ?

Reply:
I removed them by mistake. Will add it again